order/internal/server/utils.go

```diff
@@ -30,3 +33,36 @@ func (s *Server) toProtoOrderListResponse(order []models.Order) []*orderproto.Or
 	}
 	return orderResponses
 }
+
+func (s *Server) toCreateOrderEmailMessage(toEmail string, orderId int32) *types.EmailMessage {
```

Comment:
Looks like all three email messages is the same no? except the subject and body.

Maybe we can just make it a single function, and change message subject/body with an enum input or something.